content/docs/user-guide/basic-concepts/stage.md

```diff
@@ -0,0 +1,24 @@
+---
+name: 'Stage'
+match: ['stage', 'stages']
+tooltip: >-
+  A stage is a single unit of execution in a pipeline that produces some
+  (intermediate or final) output. [ð](/doc/user-guide/concepts/stage)
```

Comment:
`ð`?

Reply:
what is the current version of the tooltip? sorry, I'm lost a bit - have we migrated all of the to the new format or it's still in progress?

Reply:
There is tooltip for stage as of now. That sign was a link with a text of 📖 (like [📖 ](https://github.com/iterative/dvc.org/pull/2198) ) but why was it corrupted, I don't get. 

Updating the text and removing icon.